### ISSUE TITLE
Installs static libraries from bundled dependencies

### ISSUE THREAD

**Issue by yurivict** [urbit/urbit#992]
These shouldn't be installed:
```
Error: Orphaned: include/ed25519/ed25519.h
Error: Orphaned: include/libscrypt/libscrypt.h
Error: Orphaned: include/libscrypt/sha256.h
Error: Orphaned: include/murmur3/murmur3.h
Error: Orphaned: include/softfloat3/softfloat.h
Error: Orphaned: include/softfloat3/softfloat_types.h
Error: Orphaned: lib/libed25519.a
Error: Orphaned: lib/liblibscrypt.a
Error: Orphaned: lib/libmurmur3.a
Error: Orphaned: lib/libsoftfloat3.a
```

**Comment by ohAitch** [comment]
Huh, I thought the whole point of switching from `make` to `meson` was to use any system installed library versions, regardless of compatibility; and only fall back to fetching/building the submodules if those are nonexistent?

**Comment by yurivict** [comment]
> Huh, I thought the whole point of switching from make to meson was to use any system installed library versions, regardless of compatibility

This is unrelated. These are just wrongly-installed files. The packages themselves are either un-ported, or use legacy versions, like ```libscrypt```.

**Comment by joemfb** [comment]
Fixed in #1093
